src/libcore/mem.rs

```diff
@@ -180,6 +180,31 @@ pub fn forget<T>(t: T) {
 /// More specifically, this is the offset in bytes between successive
```

Comment:
Perhaps mention that it's always a multiple of `align_of::<T>()`?

Reply:
I don't know what you mean by this.